tests/projects/test_project.py

```diff
@@ -338,6 +338,21 @@ def test_load_project(
         assert os.path.exists(os.path.join(context, project_file))
 
 
+def test_load_project_with_setup(context):
+    url = (
+        pathlib.Path(tests.conftest.tests_root_directory)
+        / "projects"
+        / "assets"
+        / "proj-setup.zip"
+    )
+    project = mlrun.load_project(context=context, url=url)
```

Comment:
I don't understand how this test your `project_setup.py` support

Reply:
placed the context as dir under assets (vs using zip) so you can see the project files and added explanations